.github/actions/install-rust-toolchain/action.yml

```diff
@@ -0,0 +1,35 @@
+## Inspired by: https://github.com/actions-rs/toolchain/issues/216#issuecomment-1291613319
```

Comment:
With my changes below, the only thing remains the `rustup` part, which is the recommended way to install `Rust` locally (at least for development).
```suggestion
```